### ISSUE TITLE
Seems to be an issue with Angular 2 attributes

### ISSUE THREAD

**Issue by chrisnicola** [digitaltoad/vim-pug#91]
Defining attributes with `()` like this:

```
button.class((click)="onClick()")
```

Seem to cause highlighting to break for the remainder of the document.

**Comment by digitaltoad** [comment]
I haven't taken a look at any syntax changes in quite a while. I'll see what I can do.

**Comment by chrisnicola** [comment]
Interestingly it is fixed somewhat if I use `[]` instead of `()` for the outer braces.

```
button.class[(click)="onClick()"]
```

However inside those braces the highlighting is turned off completely.

**Comment by digitaltoad** [comment]
It has been a long time since I looked at Jade/Pug syntax. At one time JS was allowed in certain areas so I just used that highlighting for these areas. Some of what you are seeing is probably due to this.

**Comment by chrisnicola** [comment]
Yeah it may still be supported as well, which could present a challenge.

**Comment by NathanielInman** [comment]
I have made a pull request to fix this issue, it is available [here](https://github.com/digitaltoad/vim-pug/pull/96).

**Comment by digitaltoad** [comment]
Fixed in 22b2a486984f669027497d68c61f531306ac42ea
